### PR TITLE
`Development`: Cleanup documentation

### PR DESCRIPTION
**Pull request by 4ludwig4**
#### General

- [x] This is a small issue that I tested on the readthedocs platform. I did review it myself there.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

I found one broken link during another task and wanted to fix it.
Took this as an opportunity to also trim the line length of the other .rst files for better readability!

### Description

One commit fixes the broken links.
One commit fixes the line lengths.

### Steps for Testing

- check if links of commit one are working
- check that formatting of the modified .rst files didn't change

#### Code Review
- [x] Review 1
- [ ] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### CODE REVIEW

File: docs/dev/setup/jenkins-gitlab.rst.txt [hunk 1]
```diff
@@ -1152,21 +1159,21 @@ Prerequisites:
       Jenkins SSH Credentials
 
 8. Add a new node (select a name and select `Permanent Agent`):
```
Comment (4ludwig4) [inline docs/dev/setup/jenkins-gitlab.rst.txt:1161]: @sleiss is the indentation for step 8 and 9 intentional? I think you added this part.

Comment (4ludwig4) [reply docs/dev/setup/jenkins-gitlab.rst.txt:1161]: Just talked to Simon, and it probably wasn't intentional.

**Review by b-fein — APPROVED** [review]

**Review by tunargs — APPROVED** [review]
Link is working and formatting looks good

**Review by nikirovast — APPROVED** [review]
LGTM!

**Pull request merged**